extension/chrome/elements/pgp_pubkey.ts

```diff
@@ -67,18 +67,25 @@ Catch.try(async () => {
       if (!isUsableButExpired && ! await pubs[0].getEncryptionKey() && ! await pubs[0].getSigningKey()) {
         showKeyNotUsableError();
       } else {
+        if (compact) {
+          $('.hide_if_compact_and_not_error').remove();
+        }
+        let emailText = '';
         if (pubs.length === 1) {
           const email = pubs[0].users[0].userId ? Str.parseEmail(pubs[0].users[0].userId ? pubs[0].users[0].userId!.userid : '').email : undefined;
           if (email) {
+            emailText = email;
             $('.input_email').val(email); // checked above
-            $('.email').text(email);
           }
         } else {
-          $('.email').text('more than one person');
+          emailText = 'more than one person';
           $('.input_email').css({ display: 'none' });
           const pubToEmail = (pubkey: OpenPGP.key.Key) => Str.parseEmail(pubkey.users[0].userId ? pubkey.users[0].userId!.userid : '').email;
           Xss.sanitizeAppend('.add_contact', Xss.escape(' for ' + pubs.map(pubToEmail).filter(e => !!e).join(', ')));
         }
+        $('#pgp_block.pgp_pubkey .result')
+          .prepend(`<span>This message includes a Public Key for <span class= "email">${emailText}</span>.</span>`); // xss-direct
```

Comment:
Xss vulnerability.

Reply:
Always use the `sanitize` methods unless you have a reason not to. Don't be complacent.

Reply:
Got It,

Corrected